libs/@guardian/ab-react/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # @guardian/ab-react
 
+## 8.0.0
+
+### Major Changes
+
+- Update TS to version 5.3.3 and tslib to 2.6.2
+
+### Patch Changes
+
+- Updated dependencies
+  - @guardian/ab-core@7.0.0
+
```

Comment:
Technically, updating peer dependencies is a breaking change 😉

Reply:
Ah yeah this is Changesets trying to be helpful but actually not being that helpful. We can update the changesets config to make deps changes major by default though.